openedx_lti_tool_plugin/urls.py

```diff
@@ -9,7 +9,7 @@
         path('login', views.LtiToolLoginView.as_view(), name='lti1p3-login'),
         path('pub/jwks', views.LtiToolJwksView.as_view(), name='lti1p3-pub-jwks'),
         re_path(
-            fr'^launch/{settings.COURSE_ID_PATTERN}$',
+            fr'^launch/{settings.COURSE_ID_PATTERN}/(?P<usage_key>[^/]*)$',
             views.LtiToolLaunchView.as_view(),
             name='lti1p3-launch',
         ),
```

Comment:
```
        re_path(
            fr'^launch/{settings.COURSE_ID_PATTERN}$',
            views.LtiToolLaunchView.as_view(),
            name='lti1p3-launch',
        ),
        re_path(
            fr'^launch/{settings.COURSE_ID_PATTERN}/{settings.USAGE_KEY_PATTERN}$',
            views.LtiToolLaunchView.as_view(),
            name='lti1p3-launch',
        ),
```

@alexjmpb What do you think about using the USAGE_KEY_PATTERN settings on the pattern?.
I would also suggest doing the same on the lti-courseware view.